docs/anomaly-detection/components/README.md

```diff
@@ -21,5 +21,10 @@ This chapter describes different components, that correspond to respective secti
 - [Writer section](writer.html) - Required
 - [Monitoring section](monitoring.html) -  Optional
 
+> **Note**: starting from [v1.7.0](../CHANGELOG.md#v172), once the service starts, automated config validation is performed. Please see container logs for errors that need to be fixed to create fully valid config, visiting sections above for examples and documentation.
```

Comment:
do we have anywhere explanation of labels for this metrics, like `model` and that we copy all other labers from original timeseries, if not we need to add it somewhere

Reply:
please use abosulte pathers for link - like /anomaly-detection/change... or docs.victoriametrics.com/anomaly-detection/cha...

Reply:
nice catch, this was left from initial v1 commits, fixed

Reply:
> do we have anywhere explanation of labels for this metrics, like `model` and that we copy all other labers from original timeseries, if not we need to add it somewhere

1) https://docs.victoriametrics.com/anomaly-detection/components/writer.html#writer - right in the starting chapter (however, I will highlight it more explicitly with formatting)
2) the same page - https://docs.victoriametrics.com/anomaly-detection/components/writer.html#metrics-will also highlight it more explicitly with formatting
3) in guide - https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html#9-model-results
4) in FAQ - https://docs.victoriametrics.com/anomaly-detection/FAQ.html#what-is-anomaly-score , note section